client/components/main/menu/Menu.jsx

```diff
@@ -71,77 +112,102 @@ const Menu = ({
         <Route path="/comparison">
           <div className="menu-content">
             <h1>Comparison Filters</h1>
+            <p>* Indicates to make at least one selection</p>
+
             <InfoTitle
-              title="Date Range Selection"
+              title="Date Range Selection *"
               element="h2"
               infoText={[
                 'This filter allows the user to choose a date range for 311 comparison data.',
                 '* Please click to make a selection.',
               ]}
             />
+            {
+              (comparisonErrors.missingEndDate) ? missingSelectorWarning : null
```

Comment:
This works perfectly - no need to change the code at all. Just want to show an alternate, slightly more concise way to conditionally render in React. If you don't need to return a different component for the false condition of the ternary (i.e., returning null), this should work the same:

`{ comparisonErrors.missingEndDate && missingSelectorWarning }`

Reply:
Thank you for sharing, that's a much cleaner way to conditionally render!